integration-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# Thoth-Station Integration Tests
+
+## Managining and Creating Overlays
+
+### About our overlays
+
+Overlays are used to manage and deploy our tests for different environments. One thing to note is that comprared to most overlays for different components within `Thoth-Station`, our prod overlays (Balrog and Smuag) also includes `ocp4-stage`. This labeling format is chosen because while the cronjob itself runs on anywhere, the notification job will run in the staging cluster (OCP4). We have opted for this architecture because to send these emails, the notification service must sit behind the VPN, such as our stage cluster.
+
+### Creating overlays
+
+There is also a configmap which is used in this deployment but is not deployed from this overlay
+
+Components of the overlay:
+
+1. `integration-tests` `configMap` resource. Contains values for:
+    - `deployment-name` = The name of the deployment to the new overlay.
+    - `user-api` = The URL for user-api in the new Thoth deployment.
+    - `management-api` The URL for the management-api in the new Thoth deployment
+    - `amun-api` = The URL for the amun-api in the new Thoth deployment
+    - `tags` =
```

Comment:
tags is a filter option from integration-tests(as it is written with [behave](https://behave.readthedocs.io/en/stable/) tool).
details:https://behave.readthedocs.io/en/stable/behave.html?highlight=tags#cmdoption-t